actionkit/api/user.py

```diff
@@ -48,6 +52,73 @@ def get_user(self, user_id):
         return {'res': res,
                 'user': res.json() if res.status_code == 200 else None}
 
+    def update_user(self, user_id, update_dict):
+        res = self.client.patch(
+            #the '/' at the end is IMPORTANT!
+            '%s/rest/v1/user/%s/' % (self.base_url, user_id),
+            data=json.dumps(update_dict))
+        return self._http_return(res)
+
+    def user_search(self, **kwargs):
+        res = ''
+        if kwargs['user_id']:
+            res = self.client.get('%s/rest/v1/user/%s/' % (self.base_url, kwargs['user_id']))
+        else:
```

Comment:
Sky pointed out to me that this API is built on the requests library, so you don't have to format a query string - you can just pass a dict of search params in the .get. See https://github.com/MoveOnOrg/python-actionkit/blob/43778ef43f390869adca383816b09af2bd55a971/actionkit/api/orderrecurring.py#L31 for an example.